frontend/src/views/auth/Register.vue

```diff
@@ -5,49 +5,48 @@
       <e-text-field
         v-model="firstname"
         :name="$tc('entity.user.fields.firstname')"
+        vee-rules="required"
```

Comment:
Theoretically, vee-validate should be able to infer the required rule from the required html attribute. If this doesn't work, might be worth to dig a bit on why this is the case.

https://vee-validate.logaretm.com/v3/guide/rules.html#required

Reply:
See above, I'd propose to keep everything in `vee-rules`